{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/utils/tests.py

```diff
@@ -30,3 +31,9 @@ def test_get_site_url_negative(settings, custom_settings):
         settings.__setattr__(key, custom_settings[key])
     with pytest.raises(Exception):
         get_site_url()
+
+
+def test_password_reset_email_link(user):
+    context = user.reset_password_context()
+    html_body = get_html_body("registration/password_reset.html", context)
+    assert f"{ context['site_url'] }/password/reset/confirm/{ context['user'].id }/{ context['token'] }" in html_body
```

Comment:
This tests the other half of password reset. Making sure the URL that is generated and emailed matches our expectations.

With these two tests in place, password reset shouldn't easily break for projects. It's one of those features that no one notices is broken until a real user can't change their password.